src/main/scala/is/hail/methods/KinshipMatrix.scala

```diff
@@ -49,6 +51,93 @@ class KinshipMatrix(val hc: HailContext, val matrix: IndexedRowMatrix, val sampl
       .writeTable(output, hc.tmpDir, Some(sampleIds.mkString("\t")))
   }
 
+  def exportRel(output: String) {
+    prepareMatrixForExport(matrix).rows
+    .mapPartitions(itr => {
```

Comment:
sorry, a couple style nitpicks. I think it's nicer to write 
```scala
  .mapPartitions { itr => 
    ...
  }
```
rather than 
```scala
  .mapPartitions(itr => { 

  })
```

looks like this class could also use an autoformat, missing some spaces